### PR TITLE
Support for Rust (by @eshan-singh78)

### PR DESCRIPTION
**Pull request by SweetyAngel**

### CODE REVIEW

File: Lua/Plugins/rust.lua
Comment (RealPacket) [inline Lua/Plugins/rust.lua:1]: Please rename this file to `rs.lua`

**Review by RealPacket — COMMENTED** [review]
Rename rust.lua to rs.lua

File: Lua/Plugins/rust.lua
Comment (SweetyAngel) [reply Lua/Plugins/rust.lua:1]: another author made it

Comment (RealPacket) [reply Lua/Plugins/rust.lua:1]: > another author made it

yes, I noticed

**Pull request closed without merging**